REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.3
 Lazy
 MacroTools
+Compat
```

Comment:
If we had `@compat` usage inside, that would certainly make sense (to make the code cleaner). As it is now, Compat just adds a couple of aliases (AbstractString=String) that make Hiccup work on 0.3 as well without any work from our side. So it seems logical to just leave it for now.